wordpress-mysql-replication/website.json

```diff
@@ -138,7 +138,7 @@
                         "[concat('Microsoft.Web/Sites/', parameters('siteName'), '/config/web')]"
                     ],
                     "properties": {
-                        "packageUri": "https://auxmktplceprod.blob.core.windows.net/packages/wordpress-4.2.4-IIS.zip",
+                        "packageUri": "https://auxmktplceprod.blob.core.windows.net/packages/wordpress-4.4.2-IIS.zip",
```

Comment:
Yes it is a special package we use for web marketplace in azure portal. Using the public one  can cause issue when changes are committed to it  ( wordpress community owned). We can control the validity of the deployment in such case and we have customized to support fewer steps to configure after deployment is successful. I dont this template should be blocked due to this.